jme3-core/src/main/java/com/jme3/material/TechniqueDef.java

```diff
@@ -513,10 +514,12 @@ private Shader loadShader(AssetManager assetManager, EnumSet<Caps> rendererCaps,
             }
         }
 
-        if (getWorldBindings() != null) {
-           for (UniformBinding binding : getWorldBindings()) {
-               shader.addUniformBinding(binding);
-           }
+        final List<UniformBinding> worldBindings = getWorldBindings();
+
+        if (!worldBindings.isEmpty()) {
+            for (UniformBinding binding : worldBindings) {
```

Comment:
the empty check is not necessary here. If it's empty it won't get into the for loop anyway.
just go "for (UniformBinding binding : getWorldBindings()) {"

Reply:
I will update it